spec/controllers/concerns/administrate/punditize_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+module Administrate
+  RSpec.describe Punditize do
+    it "exists when 'Pundit' is not defined" do
+      unload_constants do
+        expect { build_dummy }.not_to raise_error
```

Comment:
Do we want to be as general as testing for all errors? Could this be more specific to the error type being experienced?

i.e. Does this catch legitimate Punditize errors?

Reply:
This is an interesting question and it is a line of thinking I went down myself.

My original test was something similar to the following.

```ruby
expect { build_dummy }.not_to raise_error LoadError
```

Rubocop was not happy with the above solution because it is too generic.

It is reasonable to expect initialising a class should not raise an error of any kind.